examples/src/main/scala/org/apache/spark/examples/ExecutorStartupTest.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.examples
+
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.SparkSession
+
+/**
+ * Simple test to distribute and return a one-item list from a Spark executor.
+ *
+ * Mainly used to measure executor startup time in dynamic allocation cluster managers like YARN
+ * and Kubernetes.
+ */
+object ExecutorStartupTest {
+  def main(args: Array[String]) {
+    val spark = SparkSession
+      .builder
```

Comment:
When you run one of these examples with spark-submit, the spark-submit script adds the various Spark config items like `spark.master` as Java System properties on the JVM.  Then the `SparkSession.builder()`

I'll update to add the `()` on the builder, though it seems to be missing on most of the examples:

```
aash@aash01-mac ~/git/spark$ grep builder examples/src/main/scala/org/apache/spark/examples/*scala
examples/src/main/scala/org/apache/spark/examples/BroadcastTest.scala:      .builder()
examples/src/main/scala/org/apache/spark/examples/DFSReadWriteTest.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/ExceptionHandlingTest.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/GroupByTest.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/HdfsTest.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/MultiBroadcastTest.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/SimpleSkewedGroupByTest.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/SkewedGroupByTest.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/SparkALS.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/SparkHdfsLR.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/SparkKMeans.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/SparkLR.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/SparkPageRank.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/SparkPi.scala:      .builder
examples/src/main/scala/org/apache/spark/examples/SparkTC.scala:      .builder
```